### ISSUE TITLE
Hiding Subpanels 

### ISSUE THREAD

**Issue by darthtang** [salesagility/SuiteCRM#802]
A cool idea I thought of was that if I wanted to hide a subpanel in say, Contacts. I would then have to remove the subpanel from view for all modules or remove the relationship.

Perhaps we can hide subpanels via studio in the subpanels component yet still keeping the relationship? 
for example;
studio -> module -> sub panel -> the sub panel -> hide this sub panel

**Comment by willrennie** [comment]
You can do this in Display Modules and Subpanels.

**Comment by darthtang** [comment]
But to hide it in a specific module? Using the Display Modules and Sub Panels would remove the sub panel completely from all modules?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Hiding Subpanels](https://trello.com/c/8nP5JSHD/127-hiding-subpanels)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
